### PR TITLE
 updated function a according to the docstring

### LINKED ISSUE #3: Do some compuation with the return value of function a
Do some computation with the return value from function a.
- Name the function using 1 letter, arguments are 1 letter, variables are 1 letter.
- Use a doc string to note what the function does.
if you need an example see `function a()`

### LINKED ISSUE #4: Create a function
create a function that is not tied to anything.
Please make the function similar to `a()`.

- Name the function using 1 letter, arguments are 1 letter, variables are 1 letter.
- Use a doc string to note what the function does.
if you need an example see `function a()`

### PR DESCRIPTION
**Pull request by rish-singhal**
Closes #3 ; closes #4

### CODE REVIEW

**Comment by rish-singhal** [comment]
Closes #4

File: src/python/main.py [hunk 1]
```diff
@@ -1,10 +1,18 @@
 #!/usr/bin/python
 
-def a(j):
+def a(j, k):
```
Comment (AllanRegush) [inline src/python/main.py]: Please remove the changes from this function. this error is on purpose :nerd_face:

**Review by AllanRegush — COMMENTED** [review]
Thank you for your PR :+1: Just a minor adjustment :nerd_face:

File: src/python/main.py (hunk 1 above)
Comment (rish-singhal) [reply src/python/main.py]: done! removed the changes

**Review by AllanRegush — APPROVED** [review]
Looks Good :+1: Thank you!

**Pull request merged**